src/main/java/com/woowacourse/zzazanstagram/config/SocketUrlMappingContext.java

```diff
@@ -0,0 +1,41 @@
+package com.woowacourse.zzazanstagram.config;
+
+import com.woowacourse.zzazanstagram.model.member.MemberSession;
+import com.woowacourse.zzazanstagram.model.member.dto.MemberResponse;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class SocketUrlMappingContext {
+    private final Map<String, MemberResponse> sessionMap = new ConcurrentHashMap<>();
+
+    public List<String> findTargetEndPoints(MemberResponse target) {
+        List<String> targets = new ArrayList<>();
+        sessionMap.entrySet().stream()
+                .filter(entry -> entry.getValue().equals(target))
+                .map(Map.Entry::getKey)
+                .forEach(targets::add);
+        return targets;
```

Comment:
```suggestion
        return sessionMap.entrySet().stream()
                .filter(entry -> entry.getValue().equals(target))
                .map(Map.Entry::getKey)
                .collect(toList());
```

stream api의 foreach는 가급적 출력 용도로만 활용해주세요.
foreach에서 로직이 수행될 경우 동시성 보장이 어려워지고 가독성이 떨어져 effective java item 46에서는 주의해서 사용하길 권하고 있습니다.